app/controllers/application_controller.rb

```diff
@@ -1,24 +1,30 @@
 class ApplicationController < ActionController::API
   include ActionController::MimeResponds
 
-
   # Defines handlers for integration with '@ember/data'
-  def self.model_accessors(model, queries: [:find, :query], limit: 10_000)
+  def self.model_accessors(
+    queries: [:find, :query],
+    includes: [],
+    filters: [],
+    limit: 10_000
```

Comment:
Is this ‘_’ intentional?

Reply:
Yup, its a "thousands" separator, to make the number easier to read:
https://en.wikibooks.org/wiki/Ruby_Programming/Syntax/Literals